jslib/fetchDataStream.js

```diff
@@ -88,7 +88,7 @@ async function fetchDataStream(dataURL, rowcb, chunkcb, endstreamcb) {
             console.error(err);
             doneLoop = true;
         });
-    };
-};
+    }
+}
```

Comment:
Same here. Adding a semicolon does not raise an error. When running `$ npm run lint:javascript`.

Reply:
yes, just saw that, can you add this to the `run-js-linter.sh` ? 

```
# Run ESLint js lib files
eslint jslib/
```